aten/src/ATen/NestedTensorImpl.cpp

```diff
@@ -153,6 +153,13 @@ c10::SymIntArrayRef NestedTensorImpl::sym_sizes_custom() const {
 c10::SymIntArrayRef NestedTensorImpl::sym_sizes() const {
   return sym_sizes_custom();
 }
+c10::SymIntArrayRef NestedTensorImpl::sym_strides_custom() const {
+  TORCH_CHECK(false, "Internal error: NestedTensorImpl doesn't support strides. Please file an issue on https://github.com/pytorch/nestedtensor");
+}
+
+c10::SymIntArrayRef NestedTensorImpl::sym_strides() const {
+  return sym_strides_custom();
```

Comment:
It's weird that we have to implement these...

Reply:
whoops this was a mistake

Reply:
whoops, this was a mistake